egs/swbd/s5c/local/chain/run_xlstm.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+# 6i is based on run_lstm_6h.sh, but changing the HMM context from triphone to left biphone.
+
+# System                       6h         6i
+# WER on train_dev(tg)      14.73      15.02
+# WER on train_dev(fg)      14.05      14.17
+# WER on eval2000(tg)        17.0       16.8
+# WER on eval2000(fg)        15.8       15.6
+# Final train prob     -0.0955829 -0.0842581
+# Final valid prob       -0.11419  -0.101598
+# Final train prob (xent)      -2.28923  -1.78616
+# Final valid prob (xent)      -2.25641  -1.79062
+# Real-time factor       3.338339   3.131303
+
+set -ex
+
+# configs for 'chain'
+stage=12
+train_stage=-1
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/lstm_6i_xconf  # Note: _sp will get added to this if $speed_perturb == true.
```

Comment:
I'd suggest 6i_xconf -> 6j or something like that.. don't want to start using _xconf in these names, we can just quietly switch over.